build/bootstrap.go

```diff
@@ -0,0 +1,20 @@
+package build
+
+// Bootstrappers reports multiaddress of bootstrap peers for a current network.
+func Bootstrappers() []string {
+	return bootstrapList[network] // network is guarantee to be valid
+}
+
+// BootstrappersFor reports multiaddress of bootstrap peers for a given network.
+func BootstrappersFor(net Network) ([]string, error) {
+	if err := net.Validate(); err != nil {
+		return nil, err
+	}
+
+	return bootstrapList[net], nil
+}
+
+// NOTE: Everytime we add a new long-running network, it's genesis hash has to be added it here.
+var bootstrapList = map[Network][]string{
+	DevNet: {}, // TODO(@Wondertan): Set multiaddress once bootstrap peers are set up.
```

Comment:
* We have to have bootstrap peers for devnet, otherwise, our DA does not exist and all the full/light nodes are living in our own world disconnected from others. This is what is happening rn with Russians. 
* As you know, diff between `devnet` and `testnet` is ephemeral and I doubt that for `testnet` we will set up everything from scratch without reusing devnet's infra. So once we start `testnet`, we will most likely just rename `DevNet` to `TestNet`.